autogen/src/header.rs

```diff
@@ -152,8 +198,7 @@ fn gen_value_enum_operand_kind(grammar: &structs::OperandKind) -> TokenStream {
             let name = as_ident(&name_str);
             let number = e.value;
             seen_discriminator.insert(e.value, name.clone());
-            enumerants.push(quote! { #name = #number });
-            from_prim_list.push(quote! { #number => Self::#name });
+            from_prim_list.push((number, name.clone()));
```

Comment:
We could rename this value to better match its intent?

Reply:
Done